pkg/pillar/cas/containerd.go

```diff
@@ -38,16 +38,13 @@ const (
 	containerRootfsPath = "rootfs/"
 	// container config file name
 	imageConfigFilename = "image-config.json"
-	// contains container's image name.
-	//nolint:unused,deadcode
-	imageNameFilename = "image-name"
```

Comment:
Any idea why this wasn't removed when the nolint comment was added? Does git blame give any hints?

Reply:
It was introduced just as a hotfix for the linter warnings, as I understand:
https://github.com/lf-edge/eve/commit/ad2f0e7e5a71b481294fd9c39b58fcf06821fd40
Maybe @deitch can provide more details.

Reply:
Yeah, it was done here:
https://github.com/lf-edge/eve/pull/3904#discussion_r1590697453

Reply:
Thanks @OhmSpectator